### PR TITLE
WT-5628 Don't perform rollback to stable on files that don't exist

### PR DESCRIPTION
**Pull request by kommiharibabu**
As part of rollback to stable ignore performing rollback to stable operation on files that don't exist and also avoid performing the checkpoint and history store search when in in_memory configuration mode

### CODE REVIEW

File: src/txn/txn_rollback_to_stable.c [hunk 1]
```diff
@@ -869,10 +872,11 @@ __wt_rollback_to_stable(WT_SESSION_IMPL *session, const char *cfg[])
     F_CLR(session, WT_SESSION_ROLLBACK_TO_STABLE);
 
     /*
-     * Forcibly log a checkpoint after rollback to stable to ensure that both in-memory and on-disk
-     * versions are same.
+     * Forcibly log a checkpoint when not in in-memory configuration after rollback to stable to
```
Comment (tammybailey) [inline src/txn/txn_rollback_to_stable.c]: Suggested rewording:
"If the configuration is not in-memory, forcibly log a checkpoint after rollback to stable to ensure that both in-memory and on-disk versions are the same."

File: src/txn/txn_rollback_to_stable.c [hunk 2]
```diff
@@ -476,11 +469,13 @@ __rollback_abort_newer_row_leaf(
             __rollback_abort_newer_insert(session, insert, rollback_timestamp);
 
         /* Abort any on-disk value. */
```
Comment (tammybailey) [inline src/txn/txn_rollback_to_stable.c]: Maybe update the comment to start with "If the configuration is not in-memory ..." ?

File: test/suite/test_rollback_to_stable04.py [hunk 3]
```diff
@@ -40,11 +41,25 @@ def mod_val(value, char, location, nbytes=1):
...
+        config = ''
+        if self.in_memory:
+            config += 'cache_size=250MB,statistics=(all),in_memory=true'
+        else:
+            config += 'cache_size=20MB,statistics=(all),log=(enabled),in_memory=false'
+        return config
+
     def test_rollback_to_stable(self):
-        nrows = 10000
+        nrows = 1000
```
Comment (tammybailey) [inline test/suite/test_rollback_to_stable04.py]: Just a FYI - I'm guessing you reduced this to make the test run faster but when I originally wrote the test I found that failures only showed up at the higher row count.

File: src/txn/txn_rollback_to_stable.c (hunk 1 above)
Comment (kommiharibabu) [reply src/txn/txn_rollback_to_stable.c]: Fixed.

File: src/txn/txn_rollback_to_stable.c (hunk 2 above)
Comment (kommiharibabu) [reply src/txn/txn_rollback_to_stable.c]: Fixed.

File: test/suite/test_rollback_to_stable04.py (hunk 3 above)
Comment (kommiharibabu) [reply test/suite/test_rollback_to_stable04.py]: Yes, I reduced the rows to improve the test speed. Now I reverted it to original value with increased cache size, so the test run time has reduced.

**Comment by kommiharibabu** [comment]
I am also observed test failures as keys are not visible after they inserted in the in-memory configuration. There may be an existing problem which got exposed to this test. I will create another ticket to handle that failure as there is no involvement of `rollback to stable`.

**Pull request merged**